README.md

```diff
@@ -47,7 +47,7 @@ gmsmith.createImages(['img1.jpg', 'img2.png'], function handleImages (err, imgs)
   canvas.addImage(imgs[1], 50, 100);
 
   // Export canvas to image
-  var resultStream = canvas['export']({format: 'png'});
+  var resultStream = canvas.export({format: 'png'});
```

Comment:
We use bracket notation to avoid linter complaints as `export` is a reserved keyword